libavfilter/vf_vpp_qsv.c

```diff
@@ -501,8 +501,8 @@ static int activate(AVFilterContext *ctx)
     VPPContext *s =ctx->priv;
     QSVVPPContext *qsv = s->qsv;
     AVFrame *in = NULL;
-    int ret, status;
-    int64_t pts;
+    int ret, status = 0;
+    int64_t pts = 0;
```

Comment:
How about to set the default pts to AV_NOPTS_VALUE ?

Reply:
Good advice.
done